Assets/HoloToolkit/Input/Tests/Scripts/ChangeColorOnFocus.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using HoloToolkit.Unity;
+using UnityEngine;
+
+/// <summary>
+/// Controls object appearance by changing its color when focused.
+/// </summary>
+public class ChangeColorOnFocus : Interactable
+{
+    [Tooltip("Object color changes to this when focused.")]
+    [SerializeField]
+    private Color focusedColor = Color.red;
+
+    private Material materialInstance;
+
+    private Color originalColor;
+
+    private void Start()
+    {
+        materialInstance = GetComponent<Renderer>().material;
```

Comment:
It's not unchanged from the original file FocuesedObjectReciever.cs
```
 -    private void Start()
 -    {
 -        material = GetComponent<Renderer>().material;
 -        originalColor = material.color;
 -
}    
```

I agree, but probably won't make this change as part of this PR.